README.md

```diff
@@ -81,22 +88,22 @@ hep.hist2dplot(H, xbins, ybins)
 # More Information
 
 ### Other styles:
-- `plt.style.use(mplhep.style.fira)` - use Fira Sans
-- `plt.style.use(style.firamath)` - use Fira Math
+- `hep.set_style("fira")` - use Fira Sans

```

Comment:
@andrzejnovak I still need to do work on this PR but I'm noticing that I'm seeing some of the font styles not being found correctly. For example, using the original `matplotlib.pyplot` API:

```python
>>> import mplhep
>>> import matplotlib.pyplot as plt
>>> plt.style.use(mplhep.style.fira)
Traceback (most recent call last):
  File "<stdin>", line 1, in <module>
AttributeError: module 'mplhep.styles' has no attribute 'fira'
```

Tab completing in the REPL shows

```python
>>> mplhep.style.
mplhep.style.ALICE       mplhep.style.CMS         mplhep.style.LHCb        mplhep.style.ROOT        mplhep.style.fabiola     mplhep.style.plt_style   mplhep.style.sys         
mplhep.style.ATLAS       mplhep.style.CMSTex      mplhep.style.LHCbTex     mplhep.style.ROOTTex     mplhep.style.firamath
```

Reply:
That's awkward. I must have removed fira when making other changes. I suggest you ignore it in this PR and I'll add it back later.

Reply:
This is now Issue #217